chat/fe/src/views/RoomView.ts

```diff
@@ -110,9 +165,113 @@ export function RoomView(props: RoomViewProps) {
       messageContentEl.append(messageTime);
 
       const bodyEl = Div();
+      setStyle(bodyEl, {
+        position: 'relative',
+      });
 
       bodyEl.innerHTML = autolinker.link(props.body);
 
+      const currentUser = getSelf();
+      if (props.user === currentUser._id) {
+        const messageOptions = Div({
```

Comment:
This is probably a good candidate for a generic component eventually.  We can play with how we build it a couple of times, but then should settle on how it finally looks/works